types/cordova-plugin-battery-status/index.d.ts

```diff
@@ -6,113 +6,26 @@
-     *                  the critical battery threshold. The value is device-specific.
-     *                  batterylow: event fires when the percentage of battery charge has
-     *                  reached the low battery threshold, device-specific value.
-     * @param callback  the function that executes when the event fires. The function is
-     *                  passed an BatteryStatusEvent object as a parameter.
-     */
-    removeEventListener(type: "batterycritical", listener: (ev: BatteryStatusEvent) => any, useCapture?: boolean): void;
-    /**
-     * Removes a listener for an event from the BatteryStatus plugin.
-     * @param type      The event to stop listening for.
-     *                  batterystatus: event fires when the percentage of battery charge
-     *                  changes by at least 1 percent, or if the device is plugged in or unplugged.
-     *                  batterycritical: event fires when the percentage of battery charge has reached
-     *                  the critical battery threshold. The value is device-specific.
-     *                  batterylow: event fires when the percentage of battery charge has
-     *                  reached the low battery threshold, device-specific value.
-     * @param callback  the function that executes when the event fires. The function is
-     *                  passed an BatteryStatusEvent object as a parameter.
-     */
-    removeEventListener(type: "batterylow", listener: (ev: BatteryStatusEvent) => any, useCapture?: boolean): void;
-    /**
-     * Removes a listener for an event from the BatteryStatus plugin.
-     * @param type      The event to stop listening for.
-     *                  batterystatus: event fires when the percentage of battery charge
-     *                  changes by at least 1 percent, or if the device is plugged in or unplugged.
-     *                  batterycritical: event fires when the percentage of battery charge has reached
-     *                  the critical battery threshold. The value is device-specific.
-     *                  batterylow: event fires when the percentage of battery charge has
-     *                  reached the low battery threshold, device-specific value.
-     * @param callback  the function that executes when the event fires. The function is
-     *                  passed an BatteryStatusEvent object as a parameter.
+     * batterylow: event fires when the percentage of battery charge has reached the low battery threshold, device-specific value.
      */
+    "batterylow": BatteryStatusEvent;
+}
+
+interface Window {
+    onbatterystatus: (type: BatteryStatusEvent) => void;
+    onbatterycritical: (type: BatteryStatusEvent) => void;
+    onbatterylow: (type: BatteryStatusEvent) => void;
+    removeEventListener<K extends keyof WindowEventMap>(type: K, listener: (ev: WindowEventMap[K]) => any, useCapture?: boolean): void;
```

Comment:
We can remove this....if we decide to add this line into `lib.dom.d.ts` https://github.com/Microsoft/TypeScript/issues/16447.... but since we are shipping soon, i think it should be ok to do it here for now